src/services/userService.js

```diff
@@ -1,11 +1,13 @@
 import { URLS } from "../constants/user";
 
+const { USERS, LOGIN, SIGNUP, GLOSSARY, GLOSSARIES, TRANSLATIONS } = URLS;
+
 export const login = async (user) => {
-  const response = await fetch(process.env.SERVER_URL + URLS.LOGIN, {
+  const response = await fetch(process.env.SERVER_URL + USERS + LOGIN, {
```

Comment:
```suggestion
  const response = await fetch(`${process.env.SERVER_URL} + ${USERS + LOGIN}`, {
```

특별한 이유가 없다면 템플릿 리터럴을 쓰는것이 좋아보이네요 !